concepts/80 Troubleshooting/45 Component Content is not Scrollable/10 The Page is Scrollable, the Component is Not.md

```diff
@@ -1,6 +1,10 @@
-If your component is not scrollable, while you can scroll the page, check the following conditions:
+If your component does not display a scrollbar, while you can scroll the page or the parent element, check the following conditions:
 
 - One of the component ancestors in the DOM has a fixed height.
```

Comment:
Maybe this should be "None of the ancestors have a fixed height"? If one of them already have a fixed height, then the resolution wouldn't work. We tell people to use fixed height for one of the parent elements (last sentence). 

```suggestion
- None of the component parents in DOM have a fixed height.
```